packages-exp/auth-exp/src/core/user/id_token_result.ts

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
Any of this need javadoc?

Reply:
yeah that's the license

Reply:
Snarky but I meant the code itself. I was just pointing to the top of the file

Reply:
Ooooooh I misunderstood, my b